src/cpp/rtps/reader/RTPSReader.cpp

```diff
@@ -137,10 +137,10 @@ SequenceNumber_t RTPSReader::update_last_notified(const GUID_t& guid, const Sequ
     return ret_val;
 }
 
-SequenceNumber_t RTPSReader::get_last_notified(const GUID_t& guid) const
+SequenceNumber_t RTPSReader::get_last_notified(const GUID_t& guid)
```

Comment:
If `const` was removed because the mutex, make it `mutable` instead of removing the `const`.

Reply:
We have to discuss this in group.